src/System.Collections/src/System/Collections/Generic/Queue.cs

```diff
@@ -309,8 +309,8 @@ public T[] ToArray()
         {
             T[] arr = new T[_size];
             if (_size == 0)
-                return arr;
-
+                return arr; // consider replacing with Array.Empty<T>() to be consistent with non-generic Queue
+            
```

Comment:
Hmm, it looks like I'm being inconsistent with myself.  I'd previously changed ```Queue.ToArray``` to return ```Array.Empty<object>```, doing so for the other NonGeneric collections as well, but I explicitly avoided doing so for the generic ones.  I don't remember now what my rationale was for thinking it was ok to convert one set but not the other.  Seems like (separate from this PR), we should reevaluate that; I'm tempted to roll back those changes to the non-generic collections, as they exist primarily for backwards compatibility, and any code more sensitive to performance should be using the generic ons, especially with value types.